File: Php/core/FormFields/FormFieldModel.php
namespace core\FormFields{

	class FormFieldModel{

		public $fieldName;
		public $value;
		public $fieldId;

		public function __construct($fieldName, $fieldId, $selectedValue){
			$this->fieldName = $fieldName;
			if($this->fieldName == null){
				$this->fieldName = '';
			}
			$this->value = $selectedValue;
			$this->fieldId = $fieldId;
		}

	}

}

File: Web/Models/FormModels/FormFieldModel.cs
namespace Web.Models.FormModels{

	public class FormFieldModel{

	    public string FieldName { get; set; }
	    public string Value { get; set; }
	    public string FieldId { get; set; }

		public FormFieldModel(string fieldName, string fieldId, string selectedValue){
			FieldName = fieldName ?? string.Empty;
			Value = selectedValue;
			FieldId = fieldId;
		}

	}

}